packages/scandipwa/src/route/MyAccount/MyAccount.container.js

```diff
@@ -361,8 +361,13 @@ export class MyAccountContainer extends PureComponent {
     }
 
     tabsFilterEnabled() {
-        return Object.fromEntries(Object.entries(MyAccountContainer.tabMap)
-            .filter(([tabName]) => MyAccountContainer.isTabEnabled(this.props, tabName)));
+        const obj = {};
+        Object.entries(MyAccountContainer.tabMap)
+            .filter(([tabName]) => MyAccountContainer.isTabEnabled(this.props, tabName))
+            // eslint-disable-next-line no-return-assign
```

Comment:
Please don't disable eslint if it's possible.
You can use the `.reduce()` method here. And you don't have to create an additional `obj` variable with it.

Reply:
done